fs_base_multi_image/__manifest__.py

```diff
@@ -18,6 +18,16 @@
         "views/fs_image.xml",
         "views/fs_image_relation_mixin.xml",
     ],
+    "assets": {
+        "web.assets_backend": [
+            "fs_base_multi_image/static/src/fields/"
+            "fs_image_relation_dnd_upload/fs_image_relation_dnd_upload.js",
```

Comment:
@benwillig To avoid the error into pre-commit, I thing you should rename your file to end with 'esm.js'.
```suggestion
            "fs_image_relation_dnd_upload/fs_image_relation_dnd_upload.esm.js",
```

Reply:
Indeed